compiler/z/codegen/OMRRegisterDependency.cpp

```diff
@@ -959,6 +959,11 @@ TR_S390RegisterDependencyGroup::assignRegisters(TR::Instruction   *currentInstru
                      break;
                   case TR_GPR:
                      opCode = TR::InstOpCode::getLoadOpCode();
+                     
+                     if (virtReg->is64BitReg())
+                        {
+                        opCode = TR::InstOpCode::LG;
```

Comment:
Is this not an issue for spill instructions, i.e. there's no corresponding change needed for ST/STG?

Reply:
I don't think so. See commit body of 55761eb for an explanation. This piece of code is very special for the way we handle spilled registers in the mainline but filled in the OOL code path. The fill portion of the puzzle happens in [1] where we check for `TR_GPR64` and correctly generate a 64-bit store to fill.

[1] https://github.com/eclipse/omr/blob/552260112c0871db8981a2b2e31f148b98c41015/compiler/z/codegen/OMRMachine.cpp#L4485-L4488